ext/php7/compatibility.h

```diff
@@ -70,6 +71,31 @@ static zend_always_inline zend_string *zend_string_init_interned(const char *str
 }
 #endif
 
+#ifdef ZEND_API_H
```

Comment:
Why is this section protected by `ZEND_API_H`?

Reply:
I believe that's an artifact of earlier code, where compatibility.h did not just include php.h, but some more select parts...